staff.rb

```diff
@@ -0,0 +1,43 @@
+require_relative 'school'
+
+class Staff < School
```

Comment:
This says "a Staff is a (kind of) School".  This sounds pretty wrong.  I would call this class `Teacher` (an apprentice teacher is a kind of teacher, or a kind of staff member, but not a kind of "Staff") and the superclass `Person`.